cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -105,6 +105,10 @@ data:
   ROOK_CSI_ENABLE_RBD: "true"
   ROOK_CSI_ENABLE_GRPC_METRICS: "false"
 
+  # Set to true to enable host networking for CSI CephFS and RBD nodeplugins.This may be necessary
+  # in some network configurations where the SDN does not provide access to an external cluster.
```

Comment:
@travisn provisioner pod is able to reach the monitor IPs in external mode and provisioner pod is already running on pod network. do we need to rephrase this one?

Reply:
Sure, what is your suggestion on the wording?

Reply:
as am thinking ,maybe if someone sees read/write performance drop with this change can revert back to hostnetworking as earlier

Reply:
@Rakshith-R still this needs to be addressed?

Reply:
``` 
  # Set to true to enable host networking for CSI CephFS and RBD nodeplugins.This may be necessary
  # in some network configurations where the SDN does not provide access to an external cluster or
  # there is significant drop in read/write performance.
```
@Madhu-1  This sounds good ?

Reply:
Looks good